qiskit_metal/qlibrary/qubits/star_qubit.py

```diff
@@ -0,0 +1,389 @@
+        * cap_style: 'round' -- round, flat, square
+        * join_style: 'round' -- round, mitre, bevel
+        * subtract: 'False'
+        * helper: 'False'
+    """
+
+    default_options = dict(radius='300um',
+                           center_radius='100um',
+                           gap_couplers='25um',
+                           gap_readout='10um',
+                           connector_length='75um',
+                           trap_offset='20um',
+                           junc_h='30um',
+                           rotation1='0.0',
+                           rotation2='72.0',
+                           rotation3='144.0',
+                           rotation4='216.0',
+                           rotation5='288.0',
+                           number_of_connectors='4',
+                           resolution='16',
+                           cap_style='round',
+                           subtract='False',
+                           helper='False')
+    """Default drawing options"""
+
+    def make(self):
+        """The make function implements the logic that creates the geometry
+        (poly, path, etc.) from the qcomponent.options dictionary of
+        parameters, and the adds them to the design, using
+        qcomponent.add_qgeometry(...), adding in extra needed information, such
+        as layer, subtract, etc.
+        """
+        p = self.p
+
+        # Extracting coordinated from the user input values
+
+        # Coordinates of the trapezoid to subtract from circle
+        coord_y1 = -p.center_radius
+        coord_y2 = -p.radius
+        coord_x1 = -p.center_radius / 2
+        coord_x2 = p.center_radius / 2
+        coord_x3 = p.center_radius + p.trap_offset
+        coord_x4 = -(p.center_radius + p.trap_offset)
+
+        # Coordinates of the polygon defining the coupling resonator (a=x, b=y)
+        coord_a1 = -(p.center_radius + p.gap_couplers)
+        coord_a2 = -p.radius
+        coord_a3 = p.radius
+        coord_b1 = -(p.center_radius / 4)
+        coord_b2 = (p.center_radius / 4)
+        coord_b3 = (p.center_radius - (p.gap_couplers))
+        coord_b4 = -(p.center_radius - (p.gap_couplers))
+        coord_b5 = -p.radius
+        coord_b6 = p.radius
+
+        # Coordinates of the polygon defining the readout resonator (c=x, d=y)
+        coord_c1 = -(p.center_radius + p.gap_readout)
+        coord_c2 = -p.radius
+        coord_c3 = p.radius
+        coord_d1 = -(p.center_radius / 2 - p.gap_readout)
+        coord_d2 = p.center_radius / 2 - p.gap_readout
+        coord_d3 = p.center_radius
+        coord_d4 = -p.center_radius
+        coord_d5 = -p.radius
+        coord_d6 = p.radius
+
+        # Connector pocket width/height
+        pocket_w = p.connector_length
+        pocket_h = p.connector_length * 2
+
+        # Connector to the JJ width/height
+        pocket_w1 = p.gap_couplers
+        pocket_h1 = p.center_radius
+
+        # create a circle
+        circle = draw.Point(p.pos_x,
```

Comment:
In the new rename we will have to change this to position_x @marcolincs

Reply:
We renamed all to pos_x, so this is consistent.